android/sound.cpp

```diff
@@ -120,38 +120,51 @@ void CSound::openStreams()
-    QString sBytesPerSample         = QString::number ( stream->getBytesPerSample() );
-    QString sBufferCapacityInFrames = QString::number ( stream->getBufferCapacityInFrames() );
-    QString sPerformanceMode        = ( stream->getPerformanceMode() == oboe::PerformanceMode::LowLatency ? "LowLatency" : "NotLowLatency" );
-    QString sSharingMode            = ( stream->getSharingMode() == oboe::SharingMode::Exclusive ? "Exclusive" : "Shared" );
-    QString sDeviceID               = QString::number ( stream->getDeviceId() );
-    QString sSampleRate             = QString::number ( stream->getSampleRate() );
-    QString sAudioFormat            = ( stream->getFormat()==oboe::AudioFormat::I16 ? "I16" : "Float" );
-    QString sFramesPerCallback      = QString::number ( stream->getFramesPerCallback() );
-    qInfo() << "Stream details: [sDirection: " << sDirection <<
-               ", FramesPerBurst: "            << sFramesPerBurst <<
-               ", BufferSizeInFrames: "        << sBufferSizeInFrames <<
-               ", BytesPerFrame: "             << sBytesPerFrame <<
-               ", BytesPerSample: "            << sBytesPerSample <<
-               ", BufferCapacityInFrames: "    << sBufferCapacityInFrames <<
-               ", PerformanceMode: "           << sPerformanceMode <<
-               ", SharingMode: "               << sSharingMode <<
-               ", DeviceID: "                  << sDeviceID <<
-               ", SampleRate: "                << sSampleRate <<
-               ", AudioFormat: "               << sAudioFormat <<
-               ", FramesPerCallback: "         << sFramesPerCallback << "]";
+    QString sDirection =
+        ( stream->getDirection() == oboe::Direction::Input ? "Input"
+                                                           : "Output" );
+    QString sFramesPerBurst = QString::number ( stream->getFramesPerBurst() );
+    QString sBufferSizeInFrames =
+        QString::number ( stream->getBufferSizeInFrames() );
+    QString sBytesPerFrame  = QString::number ( stream->getBytesPerFrame() );
+    QString sBytesPerSample = QString::number ( stream->getBytesPerSample() );
+    QString sBufferCapacityInFrames =
+        QString::number ( stream->getBufferCapacityInFrames() );
+    QString sPerformanceMode =
+        ( stream->getPerformanceMode() == oboe::PerformanceMode::LowLatency
+              ? "LowLatency"
+              : "NotLowLatency" );
+    QString sSharingMode =
+        ( stream->getSharingMode() == oboe::SharingMode::Exclusive ? "Exclusive"
+                                                                   : "Shared" );
+    QString sDeviceID   = QString::number ( stream->getDeviceId() );
+    QString sSampleRate = QString::number ( stream->getSampleRate() );
+    QString sAudioFormat =
+        ( stream->getFormat() == oboe::AudioFormat::I16 ? "I16" : "Float" );
+    QString sFramesPerCallback =
+        QString::number ( stream->getFramesPerCallback() );
```

Comment:
I think widening will solve a lot.  But the wrapping rules still need to be "right".  Line 142 is actually wrongly written - either the `(`/`)` pair aren't needed or should surround the boolean expression, not the whole expression - but we're not getting that kind of correction "for free" ;).  Line 144 is probably correctly wrapped - if it were long enough.

I don't know what the line width for wrapping is set to, but I'd aim for 120 minimum and then we can try increasing in steps until we're happy.  (Yes, I saw you said something about that, @hoffie ;) )

Reply:
ColumnLimit=120
https://github.com/passing/jamulus/commit/b50bafc55cbd1ec253a51b4065b07aa16d2babf0#diff-d5918572149dc4cba14e4f956ad7f44910f17eb8d402a3f95fc3475903280e5bL123-R138
ColumnLimit=1280
https://github.com/passing/jamulus/commit/b50bafc55cbd1ec253a51b4065b07aa16d2babf0#diff-b6ae5abb1f10f614aa06aadc13071f22937d5be4de59b2c5ce14c05b694d11e3L123-R131

Reply:
> Indentations in preprocessor macros are dropped. Check if they can be kept somehow

having them exactly like they were is not possible, but could be changed to one of these two options:
https://github.com/passing/jamulus/blob/b50bafc55cbd1ec253a51b4065b07aa16d2babf0/test_pp_before_hash/src/server.cpp#L35-L42
https://github.com/passing/jamulus/blob/b50bafc55cbd1ec253a51b4065b07aa16d2babf0/test_pp_after_hash/src/server.cpp#L35-L42

Reply:
> I don't know what the line width for wrapping is set to, but I'd aim for 120 minimum ...

totally agree it should be 120 (or more)
The code from above would then look like this:
https://github.com/passing/jamulus/blob/b50bafc55cbd1ec253a51b4065b07aa16d2babf0/test_120/android/sound.cpp#L118-L139

going to 180 would result in
https://github.com/passing/jamulus/blob/b50bafc55cbd1ec253a51b4065b07aa16d2babf0/test_180/android/sound.cpp#L114-L132
so in this situation it would even align all the assignments - but it just gets pretty wide then

Reply:
Thanks! At least in this case the latter (180) looks more readable to me. Let's see what @pljones thinks.